sw/host/opentitantool/src/command/lc.rs

```diff
@@ -282,7 +305,22 @@ impl CommandDispatch for Transition {
             .create(transport)?
             .connect(JtagTap::LcTap)?;
 
+        // In order to be on the safe side, we're asserting ROM bootstrap and
+        // reset the chip to prevent ROM from going into a reset loop.
+        let rom_bootstrap = transport.pin_strapping("ROM_BOOTSTRAP")?;
+        rom_bootstrap.apply()?;
+
+        // Reset the chip so that LC_CTRL is in a clean state.
+        let _ = transport.reset_target(Duration::from_millis(50), false);
+        std::thread::sleep(Duration::from_millis(50));
+
+        // Check whether this is a valid transition.
         let token_words = parse_token_str(self.token.as_str())?;
+        check_lc_transition(
+            &mut *jtag,
+            self.target_lc_state,
+            DifLcCtrlToken::from(token_words),
```

Comment:
Can you make `parse_token_str` return `DifLcCtrlToken` instead?

Reply:
Yes, done.